tests/e2e/performance/csi_tests/test_pvc_snapshot_performance.py

```diff
@@ -303,33 +383,54 @@ def test_pvc_snapshot_performance(self, teardown_factory, pvc_size):
 
             all_results.append(test_results)
 
-        # logging the test summery, all info in one place for easy log reading
+        # logging the test summary, all info in one place for easy log reading
         c_speed, c_runtime, r_speed, r_runtime = (0 for i in range(4))
-        log.info("Test summery :")
+
+        log.info("Test summary :")
         for tst in all_results:
             c_speed += tst["create"]["speed"]
             c_runtime += tst["create"]["time"]
             r_speed += tst["restore"]["speed"]
             r_runtime += tst["restore"]["time"]
+            self.full_results.all_results["creation_time"].append(tst["create"]["time"])
+            self.full_results.all_results["creation_speed"].append(
+                tst["create"]["speed"]
+            )
+            self.full_results.all_results["restore_time"].append(tst["restore"]["time"])
+            self.full_results.all_results["restore_speed"].append(
+                tst["restore"]["speed"]
+            )
+            self.full_results.all_results["dataset_inMiB"] = tst["dataset"]
             log.info(
                 f"Test {tst['test_num']} results : dataset is {tst['dataset']} MiB. "
                 f"Take snapshot time is {tst['create']['time']} "
                 f"at {tst['create']['speed']} MiB/Sec "
                 f"Restore from snapshot time is {tst['restore']['time']} "
                 f"at {tst['restore']['speed']} MiB/Sec "
             )
-        log.info(
-            f" Average snapshot creation time is {c_runtime / self.tests_numbers} sec."
-        )
-        log.info(
-            f" Average snapshot creation speed is {c_speed / self.tests_numbers} MiB/sec"
-        )
-        log.info(
-            f" Average snapshot restore time is {r_runtime / self.tests_numbers} sec."
-        )
-        log.info(
-            f" Average snapshot restore speed is {r_speed / self.tests_numbers} MiB/sec"
-        )
+
+        avg_snap_c_time = c_runtime / self.tests_numbers
+        avg_snap_c_speed = c_speed / self.tests_numbers
+        avg_snap_r_time = r_runtime / self.tests_numbers
+        avg_snap_r_speed = r_speed / self.tests_numbers
+        log.info(f" Average snapshot creation time is {avg_snap_c_time} sec.")
+        log.info(f" Average snapshot creation speed is {avg_snap_c_speed} MiB/sec")
+        log.info(f" Average snapshot restore time is {avg_snap_r_time} sec.")
+        log.info(f" Average snapshot restore speed is {avg_snap_r_speed} MiB/sec")
+
+        self.full_results.add_key("avg_snap_creation_time_insecs", avg_snap_c_time)
+        self.full_results.add_key("avg_snap_creation_speed", avg_snap_c_speed)
+        self.full_results.add_key("avg_snap_restore_time_insecs", avg_snap_r_time)
+        self.full_results.add_key("avg_snap_restore_speed", avg_snap_r_speed)
+
+        # Write the test results into the ES server
+        log.info("writing results to elastic search")
```

Comment:
Please consider changing the print to : 
 log.info("writing results to elastic search server")